src/scss/skin-modern/components/_subtitleoverlay.scss

```diff
@@ -1,7 +1,7 @@
 @import '../variables';
 @import '../mixins';
 
-.#{$prefix}-ui-subtitle-overlay {
+.#{$prefix}-ui-uicontainer .#{$prefix}-ui-subtitle-overlay {
```

Comment:
Why do we need to scope this to the `-ui-uicontainer` now? 🤔 Where else should the `-ui-subtitle-overlay` be? 🤔

Reply:
It adds more prio score for these style definitions if there is `*` selector that would have higher priority.

Reply:
Hmm I see 🤔 But why would we only need it on the `subtitle-overlay` and not on all other components as well? 🤔